examples/terraform-aws-ssm-example/variables.tf

```diff
@@ -0,0 +1,3 @@
+variable "region" {
+  type = string
```

Comment:
Please add a `description` and `default`. See also all the other `variables.tf` files for the typical comments/structure we have in these files.